hail/src/main/scala/is/hail/expr/ir/EmitStream.scala

```diff
@@ -651,8 +658,11 @@ object EmitStream {
       }
 
       streamIR match {
-        case NA(_) =>
-          COption.None
+        case x@NA(_) =>
+          COption.none(SizedStream(
+            Code._empty,
+            coerce[PCanonicalStream](x.pType).defaultValue.stream,
+            Some(0)))
```

Comment:
having a length for a NA stream is a bit weird, but I think I understand why it's here.

Reply:
It just has to create some `SizedStream`, which must have no observable effect on the generated code. Making it have length 0 made some tests easier, but otherwise it should be irrelevant.